moviepy/audio/fx/audio_normalize.py

```diff
@@ -19,4 +20,9 @@ def audio_normalize(clip):
     """
 
     max_volume = clip.max_volume()
-    return volumex(clip, 1 / max_volume)
+    if max_volume == 0:
```

Comment:
Why not just `max_volume or 1`? In that case, if is `0` the `1` will be used: 

```suggestion
    return volumex(clip, 1 / max_volume or 1)```

Reply:
Not sure about your suggestion here... it would be shorter, but I don't think it is easier to read, and potentially worse performing, because it adds an extra layer to the fx stack and multiplies every audio frame by 1 every frame unnecessarily. Of course without actually benchmarking it, it might have negligible impact after all.

Reply:
:thinking: Maybe I'm not clear with the suggestion... I mean this:

```suggestion
   return volumex(clip, 1 / (max_volume or 1))```
```

Removing all the lines:

```python
    if max_volume == 0:
        # Nothing to normalize.
        # Avoids a divide by zero error.
        return clip.copy()
    else:
        return volumex(clip, 1 / max_volume)

Reply:
No I understood that. I meant that it would be worse performing only when max_volume == 0 because it calls `volumex` unnecessarily in that situation.
I don't have a strong opinion so perhaps a comment to explain the `or 1`?

Reply:
Oh yes, my bad :man_facepalming: